src/javafundamentals/interfaces/Ex01/CarsForSale.java

```diff
@@ -0,0 +1,13 @@
+package javafundamentals.interfaces.Ex01;
+
+public class CarsForSale implements SalesMethods{
+    @Override
+    public void onlinePhotos() {
+        System.out.println("Car photos on craigslist");
+    }
+
+    @Override
+    public void signOnObject() {
```

Comment:
The same comment about method names should be verbs

Reply:
thanks!